skiplang/prelude/ts/src/sk_types.ts

```diff
@@ -52,6 +52,10 @@ class SkError extends Error {
   }
 }
 
+export function fail(msg: string): never {
+  throw new Error(msg);
+}
+
```

Comment:
This looks suspicious: `Error` is a native JS class, so the `fail()` function doesn't have much to do with skiplang's standard library.

Reply:
Yeah, fair enough. I'll look for a better place. I use this in the prelude, the ts examples, and skdb ts code, so it isn't obvious to me where it should go. Really it is just a work-around for the fact that `throw new Error(...)` is not allowed as the RHS of `??` while a function call is. I think I could instead replace every call to `fail` with something like `(() => throw new Error(...))()`. WDYT? (Grr, languages that don't get expressions right.)